ocular/src/keyring.rs

```diff
@@ -31,7 +32,7 @@ pub trait KeyStore {
     fn key_exists(&self, keyname: &KeyName) -> Result<bool, KeyStoreError>;
 
     /// Add a private key document associated with a key name into the keystore.
-    fn add_key(
+    fn create_key(
```

Comment:
sorry i should have been clearer, add_key makes sense here because the key exists and we're adding it. for the _generated_mnemonic one i think we need add_

Reply:
Ah i gotchu, no worries. Updated.